prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```diff
@@ -171,7 +171,7 @@ class FHIRConverter(
                         val blobInfo = BlobAccess.uploadBody(
                             Report.Format.FHIR,
                             bodyBytes,
-                            report.name,
+                            report.id.toString(),
```

Comment:
I left a comment earlier that `report.name` isn't used anymore so maybe it could be just repurposed to be `report.id.toString()`?